pipelines/build/common/openjdk_build_pipeline.groovy

```diff
@@ -906,6 +906,15 @@ class Build {
                 }
             }
 
+            try {
```

Comment:
Doesn't this contravene the `cleanWorkspace` flag?

Also, assuming this was OK to execute, will this fail on Windows as per the special case logic above?

Reply:
clean Workspace is different, it remove the "whole" Jenkins workspace ie.removes the existing openjdk-build repo and it's sub openjdk src repo.
This removes just the workspace/build/src/build folder to aid the subsequent git clean -fdx which always gets called to remove non-git files.

Reply:
The Windows special case logic is a workaround for the Process locked files issue.